src/store/offerPreferenes/useOfferPreferences.ts

```diff
@@ -20,7 +20,8 @@ export type SellSorter = "highestPrice" | "bestReputation";
 type OfferPreferences = {
   buyAmountRange: [number, number];
   sellAmount: number;
-  premium: number;
+  premium: number; // for Sell
+  buyPremium: number;
```

Comment:
wdyt about just calling this "maxPremium" instead? don't really mind it this way either though